naboj/Makefile

```diff
@@ -0,0 +1,123 @@
+.SECONDARY:
+
+version =           '1.26'
```

Comment:
To neviem. Samozrejme som to tak chcel mať, ale v mnohých smeroch robí úplne odlišné veci (napríklad sa píše a bude písať rovno v TeXu, robí kopu nezávislých vecí typu tearoff alebo obálky). Plus to celé bolo robené dosť narýchlo na chate v lete.

Porozmýšľam nad tým, ale nesľubujem, že to pôjde ľahko.